pkgs/applications/graphics/glimpse/plugins/default.nix

```diff
@@ -0,0 +1,215 @@
+        url = http://registry.gimp.org/files/wavelet-sharpen-0.1.2.tar.gz;
+        sha256 = "0vql1k67i21g5ivaa1jh56rg427m0icrkpryrhg75nscpirfxxqw";
+      };
+      installPhase = "installPlugins src/wavelet-sharpen"; # TODO translations are not copied .. How to do this on nix?
+    };
+
+    lqrPlugin = pluginDerivation {
+      # menu:
+      # Layer/Liquid Rescale
+      name = "lqr-plugin-0.6.1";
+      buildInputs = with pkgs; [ liblqr1 ];
+      src = fetchurl {
+        url = http://registry.gimp.org/files/gimp-lqr-plugin-0.6.1.tar.bz2;
+        sha256 = "00hklkpcimcbpjly4rjhfipaw096cpy768g9wixglwrsyqhil7l9";
+      };
+      #postInstall = ''mkdir -p $out/nix-support; echo "${liblqr1}" > "$out/nix-support/propagated-user-env-packages"'';
+      installPhase = "installPlugins src/gimp-lqr-plugin";
+    };
+
+    gmic = pkgs.gmic-qt.override {
+      variant = "gimp";
+      gimp = glimpse;
+    };
+
+    ufraw = (
+      pkgs.ufraw.override {
+        gimp = glimpse;
+      }
+    ).gimpPlugin;
+
+    gimplensfun = pluginDerivation rec {
+      version = "unstable-2018-10-21";
+      name = "gimplensfun-${version}";
+
+      src = fetchFromGitHub {
+        owner = "seebk";
+        repo = "GIMP-Lensfun";
+        rev = "1c5a5c1534b5faf098b7441f8840d22835592f17";
+        sha256 = "1jj3n7spkjc63aipwdqsvq9gi07w13bb1v8iqzvxwzld2kxa3c8w";
+      };
+
+      buildInputs = with pkgs; [ lensfun gexiv2 ];
+
+      installPhase = "
+      installPlugins gimp-lensfun
+    ";
+
+      meta = {
+        description = "GIMP plugin to correct lens distortion using the lensfun library and database";
+
+        homepage = http://lensfun.sebastiankraft.net/;
+
+        license = stdenv.lib.licenses.gpl3Plus;
+        maintainers = [ ];
+        platforms = stdenv.lib.platforms.gnu ++ stdenv.lib.platforms.linux;
+      };
+    };
+
+    # =============== simple script files ====================
+
+    lightning = scriptDerivation {
+      name = "Lightning";
+      src = fetchurl {
+        url = http://registry.gimp.org/files/Lightning.scm;
+        sha256 = "c14a8f4f709695ede3f77348728a25b3f3ded420da60f3f8de3944b7eae98a49";
+      };
+    };
+
+    # space in name trouble ?
+
+    rainbowPlasma = scriptDerivation {
+      # http://registry.gimp.org/node/164
+      name = "rainbow-plasma";
+      src = fetchurl {
+        url = "http://registry.gimp.org/files/Rainbow Plasma.scm";
+        sha256 = "34308d4c9441f9e7bafa118af7ec9540f10ea0df75e812e2f3aa3fd7b5344c23";
+        name = "Rainbow-Plasma.scm"; # nix doesn't like spaces, does it?
+      };
+    };
+
+    # doesn't seem to be working :-(
+    lightningGate = scriptDerivation {
+      # http://registry.gimp.org/node/153
+      name = "lightning-gate";
+      src = fetchurl {
+        url = http://registry.gimp.org/files/LightningGate.scm;
+        sha256 = "181w1zi9a99kn2mfxjp43wkwcgw5vbb6iqjas7a9mhm8p04csys2";
+      };
+    };
+
+  } // stdenv.lib.optionalAttrs (config.allowAliases or true) {
```

Comment:
Same for these legacy aliases and the focusblur patch file.

Reply:
Done. I've never seen `allowAliases` before, is this even available as a nixpkgs config option? I removed the whole attrs.

Reply:
Yes, you can set it like any other [global config](https://nixos.org/manual/nixpkgs/unstable/#chap-packageconfig). It is currently undocumented but ofborg and some other parts of infrastructure disable them, and we recommend people to disable them as well.

Reply:
Good to know. Would be nice to have it discoverable like `nixpkgs.config.allowUnfree`.